docs/developers/cookbook.rst

```diff
@@ -0,0 +1,239 @@
+Cheat Sheet
+===========
+
+For most operations below, you need to create a :py:class:`DataObsClient <cartoframes.data.clients.DataObsClient>` object. For example, here's how user `cyclingfan` with API key `abc123` creates one:
+
+.. code::
+
+    from cartoframes.auth import Credentials
+    from cartoframes.data.clients import DataObsClient
+
+    credentials = Credentials(username='cyclingfan', api_key='abc123')
+    do = DataObsClient(credentials)
+
+How to get census tracts or counties for a state
+------------------------------------------------
+
+It's a fairly common use case that someone needs the Census tracts for a region. With cartoframes you have a lot of flexibility for obtaining this data.
+
+1. Get bounding box of the region you're interested in. Tools like `Klockan's BoundingBox tool <https://boundingbox.klokantech.com/>`__ with the CSV output are prefect. Alternatively, use a table with the appropriate covering region (e.g., an existing table with polygon(s) of Missouri, its counties, etc.).
+2. Get the FIPS code for the state(s) you're interested in. The US Census `provides a table <https://www.census.gov/geo/reference/ansi_statetables.html>`__ as do many other sites. In this case, I'm choosing ``29`` for Missouri.
+
+.. code::
+
+    from cartoframes.data import Dataset
+    from cartoframes.viz import Map, Layer
+
+
+    # get all census tracts (clipped by water boundaries) in specific bounding box
+    missouri_ct_dataset = do.boundaries(
+        region=[-95.774147,35.995682,-89.098846,40.613636],
+        boundary='us.census.tiger.census_tract_clipped'
+    )
+
+    # get dataframe
+    df = missouri_ct_dataset.dataframe
+
+    # filter out all census tracts that begin with Missouri FIPS (29)
+    # GEOIDs begin with two digit state FIPS, followed by three digit county FIPS
+    missouri_ct_dataframe = df[df.geom_refs.str.startswith('29')]
```

Comment:
Maybe we can add the visualization step with the dataframe too, which actually works without the upload.